superset/migrations/versions/d416d0d715cc_add_limiting_factor_column_to_query_.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_limiting_factor_column_to_query_model.py
+
+Revision ID: d416d0d715cc
+Revises: 19e978e1b9c3
+Create Date: 2021-04-16 17:38:40.342260
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "d416d0d715cc"
+down_revision = "19e978e1b9c3"
+
+from enum import Enum
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.dialects.postgresql.base import PGDialect
+
+limiting_factor = postgresql.ENUM(
```

Comment:
This is specific for postgres, we support MySQL also and SQLite (for now). Also we tend to not use ENUM's types on the database since they can quickly require migrations when a new enum value is needed

Reply:
If we use a Varchar here instead, that should suffice. SqlA can then serialize Python enum vals into its string repr. This would work across all DB types :)

Reply:
hello! I ended up using a varchar, I think it is better!